README.md

```diff
@@ -276,6 +276,30 @@ func HandleRequest(ctx context.Context, name string) (string, error) {
 }
 ```
 
+`aws-xray-sdk-go` doesn't currently support streaming grpc call.
+
+**grpc client**
+
+```go
+conn, err := grpc.Dial(
```

Comment:
Can we add a little description here on how below grpc client and server instrumentation works ?

Reply:
Added a short description to point out the instrumentation is done by applying `xray.UnaryServerInterceptor` or `xray.UnaryClientInterceptor`.